ironic-config/ironic.conf.j2

```diff
@@ -100,7 +102,7 @@ disable_deep_image_inspection = True
 {% if env.IRONIC_USE_MARIADB | lower == "true" %}
 connection = {{ env.MARIADB_CONNECTION }}
 {% else %}
-connection = sqlite:////var/lib/ironic/ironic.sqlite
+connection = {{ env.LOCAL_DB_URI }}
```

Comment:
nit: we should finally decide whether we prefix variables with IRONIC_ or not :)